source/simulator/core.cc

```diff
@@ -1976,6 +1976,26 @@ namespace aspect
           break;
         }
 
+        case NonlinearSolver::Binary_input:
+        {
+          // Identical to IMPES except does not solve Stokes equation
+          if (parameters.free_surface_enabled)
+            free_surface->execute ();
+       
+          std::string fileName = parameters.input_directory + "/" + "solution-" + Utilities::int_to_string(timestep_number, 5) + ".mesh";
+          triangulation.load(fileName.c_str());
+          parallel::distributed::SolutionTransfer<dim, LinearAlgebra::BlockVector> sol_trans(dof_handler);
+          sol_trans.deserialize (solution);
+          //triangulation.clear();
```

Comment:
Originally I began implementing the reading in and swapping of solution data as an extension of the prescribed stokes plugin. However, I realized that:

1. A child of the prescribed stokes solution interface cannot modify the solution block vectors
2. Uncertain as to how to evaluate the velocity and pressure at a given position using the read in solution

Therefore, I ended up adding a new Nonlinear solver called binary_input to circumvent these issues. Is this the proper place for this feature or is the extension of the prescribed stokes plugin the right approach?